view-user/src/pages/index.tsx

```diff
@@ -80,20 +80,7 @@ const Page: NextPage = () => {
           </div>
         </div>
       </Modal>
-      <Header user="">
-        <div className={styles.main}>
-          <Button
-            size="m"
-            shape="circle"
-            onClick={() => router.push("./prizes")}
-          >
-            <div className={styles.buttonContents}>
-              <Image src="/GiftBox.svg" alt="GiftBox" width={19} height={19} />
-              {t.PRIZE_BUTTON}
-            </div>
-          </Button>
-        </div>
-      </Header>
+      <Header></Header>
```

Comment:
```suggestion
      <Header />
```
これは人による。
何もない時はこうした方が良いかも